index.html

```diff
@@ -421,6 +421,110 @@ <h1 class="w-full my-2 text-4xl font-bold leading-tight text-center text-gray-80
               </div>
             </a>
         </div>
+      </div>
+    </section>
+
+    <!--Testimonial-->
+    <section class="bg-gray-50 py-8">
+      <div class="container max-w-5xl mx-auto m-8">
+        <h1 class="w-full my-2 text-4xl font-bold leading-tight text-center text-gray-800">
+          Testimonial
+        </h1>
+        <p class="text-center text-gray-800 text-base px-6 mb-5 mx-10">
+          Designer and developer using the best tools
```

Comment:
Can we change to

`Community Members Experience`

Reply:
Yes I think it'll be good